app/models/product.rb

```diff
@@ -13,7 +13,7 @@ class Product < ActiveRecord::Base
     },
     s3_host_name: ENV['S3_HOST_NAME'],
     s3_protocol: "",
-    styles: {preview: '50x50#', thumbnail: '215x133#'}
+    styles: {preview: '50x50#', show: '584x399#', thumbnail: '215x133#'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.